src/app/shared/learning-object/learning-object.component.scss

```diff
@@ -1,6 +1,9 @@
 @import '~_vars.scss';
 
 @media (min-width: 751px) {
+    .toolTip { 
+        text-transform: lowercase;
```

Comment:
Why are we displaying text content lowercased inside the tooltip? Also, why is that logic dependent on the size of the screen?

Reply:
I didn't mean to put that inside the @media scope but this is how I am able to lowercase the 'and' and 'other' words sense they were automatically being converted to titlecase